components/BurgerMenuNav.vue

```diff
@@ -59,159 +26,77 @@ export default {
+.navigation-slice-zone {
+  .bm-menu {
+    background: linear-gradient(134.72deg, $blue-2 0%, $blue-1 100%);
 
-    p {
-      margin: 0;
+    @include device-is('large-screen') {
+      display: none;
     }
-  }
 
-  hr {
-    height: 1px;
-    opacity: 0.5;
-    border: 1px solid $grey-7;
-    -webkit-border-radius: 0px;
-    -moz-border-radius: 0px;
-    border-radius: 0px;
-    display: inline-block;
-  }
-  .nav-middle {
-    font-family: $font-roboto;
-    h3 {
-      color: $white;
-      font-size: 16px;
-      font-weight: 700;
-      margin-top: 17.5px;
-    }
-    hr {
-      width: 16px;
-    }
     ul {
-      margin-left: 24px;
-    }
-    &__link {
-      position: relative;
-      margin-bottom: 14px;
-      a {
-        font-size: 14px;
-      }
-    }
-    .nuxt-link-active {
-      font-weight: 700;
-    }
-    .nuxt-link-active:before {
-      position: absolute;
-      left: -48px;
-      content: ' ';
-      height: 100%;
-      width: 2px;
-      background-color: $white;
-    }
-  }
+      padding-left: 0;
+      list-style: none;
 
-  .nav-bottom {
-    &__link {
-      font-weight: 400;
-      color: $white;
-      text-align: left;
-      margin-top: 10px;
-    }
+      & > li {
+        padding-left: 0;
 
-    .show-page-icon {
-      font-size: 10px;
-      margin-left: 5px;
+        &::before {
+          display: none;
+        }
+      }
     }
 
-    &__bar {
-      margin-bottom: 12px;
+    .nav-top {
+      &__link {
+        font-weight: 600;
+        color: $white;
+        text-align: left;
+        margin-bottom: 12px;
+
+        a {
+          text-decoration: none;
+          display: block;
+          transition: 0.3s;
+          font-size: 1rem;
+          line-height: 1.5rem;
+          color: $white;
+
+          &:hover {
+            text-decoration: underline;
+          }
+        }
+      }
     }
 
-    hr {
-      width: 180px;
-    }
+    .bm-item-list {
+      height: 100%;
+      padding: 0 25px;
 
-    a {
-      text-decoration: none;
-      display: flex;
-      align-items: baseline;
-      transition: 0.3s;
-      font-size: 12px;
-      line-height: 17px;
-      color: $white;
+      & > * {
+        padding: 0;
+        flex-direction: column;
+        justify-content: space-between;
+        height: 100%;
+      }
     }
   }
 
-  .nav-top a:hover,
-  .nav-bottom a:hover {
-    text-decoration: underline;
+  .bm-burger-bars {
+    background-color: $grey-1;
   }
 
-  .bm-item-list {
-    height: 100%;
-    margin-left: 0;
-    padding: 0 25px;
+  .bm-burger-button {
+    top: 25px;
 
-    & > * {
-      padding: 0;
-      flex-direction: column;
-      justify-content: space-between;
-      height: 100%;
+    @include device-is('large-screen') {
+      display: none;
     }
   }
-}
-
-.bm-burger-bars {
-  background-color: $grey-1;
-}
 
-.bm-burger-button {
-  @include device-is('desktop') {
-    display: none;
+  .bm-cross {
+    background: $grey-1;
+    height: 24px !important;
```

Comment:
En général, `!important` est un bad smell. Comment est-ce qu'on peut l'enlever ?
```suggestion
    height: 24px;
```

Reply:
Je t'avoue ne pas avoir vu le `important` qui trainait.
En regardant de plus près, le `important` a été rajouté car le composant `vue-burger-menu` fait des manipulations directement sur l'élément sans passer par une classe css.
Je propose de voir cela lors de la refonte du burger-menu.